talent/job-descriptions/offers/infrastructure.md

```diff
@@ -13,7 +13,7 @@ Software Engineer in the Infrastructure team
 
 The Infrastructure team focuses on managing and growing our bare metal computation cluster. This cluster allows us to run large scale computational algorithms and store the required information for those computations.
 
-- Our cluster, with close to 2000 threads, more than 11TB of RAM and 1PB of storage, is backed by CoreOS and Kubernetes. It has two main goals: on the one hand, it stores all the available public code in a HDFS cluster and on the other hand, it runs intensive computation jobs over the stored data on top of Spark. 
+- Our cluster, with close to 2000 threads, more than 11TB of RAM and 1PB of storage, is backed by CoreOS and Kubernetes. It has two main goals: on the one hand, it stores all the available public code in Gluster nodes and on the other hand, it runs intensive computation jobs over the stored data on top of Spark. 
```

Comment:
We're not even sure that Gluster is going to be the definitive solution. We can just leave in "stores all the available public code in a distributed filesystem"